understanding/motion-actuation.html

```diff
@@ -31,77 +31,76 @@ <h1>達成基準 2.5.4: 動きによる起動を理解する</h1>
          <dl>
-            
-            
+
+
             <dt>サポートされたインタフェース</dt>
-            
+
             <dd><a href="https://w3c.github.io/wcag/guidelines/#dfn-accessibility-supported" target="terms">アクセシビリティ サポーテッド</a>なインタフェースを通じて機能を操作するために動きが用いられる。</dd>
-            
-            
+
+
             <dt>必要不可欠</dt>
-            
+
             <dd>その機能にとって動きが<a href="https://w3c.github.io/wcag/guidelines/#dfn-essential" target="terms">必要不可欠</a>であり、この達成基準に従うと動作を無効化してしまう。</dd>
-            
-            
+
+
          </dl>
       </blockquote>
       <main>
          <section id="intent">
             <h2>意図</h2>
-            <p>この達成基準の目的は、（カメラのようなセンサーが握ってジェスチャーを解釈できるように）デバイスを動かすこと（たとえば、振ることや傾けること）によって、またはデバイスに向かってジェスチャーすることによって引き起こされる機能を確実にすることである。 動きが機能にとって不可欠であるか、または動きまたはジェスチャーを使用しないことが活動を無効にしない限り、より従来のユーザインターフェースコンポーネントによって操作される。</p>
+            <p>この達成基準の目的は、 (カメラのようなセンサーがジェスチャーを取得して解釈できるように) デバイスを動かすこと (たとえば、振ることや傾けること) によって、又はデバイスに向かってジェスチャーすることによって引き起こされる機能を確実にすることである。動きが機能にとって不可欠であるか、又は動きもしくはジェスチャーを使用しないことが活動を無効にしない限り、より従来のユーザインターフェースコンポーネントによって操作される。</p>
             <div class="note"><div role="heading" class="note-title marker" aria-level="3">注記</div>
-               <p>この基準は、デバイスに向かってジェスチャーする、デバイスを傾ける、またはデバイスを振るなどの動作に直接反応するセンサを介した入力に関するものである。 それは、地理的位置センサまたはビーコンによって記録されるような空間を通過するユーザの動き、またはユーザによる意図的なジェスチャ以外のデバイスで観察されるイベントをカバーしない。 また、キーボード、ポインター、または支援技術の操作に関連した間接的な動作についてもカバーしない。</p>
+               <p>この基準は、デバイスに向かってジェスチャーする、デバイスを傾ける、又はデバイスを振るなどの動作に直接反応するセンサーを介した入力に関するものである。 この基準は、地理的位置センサー、又はビーコンによって記録されるような空間を通過する利用者の動き、又は利用者による意図的なジェスチャー以外のデバイスで観察されるイベントを取り扱う。 また、キーボード、ポインタ、又は支援技術の操作に関連した間接的な動作についてもカバーしない。</p>
             </div>
-            <p>デバイスには、電話やタブレットデバイスの加速度センサーやジャイロセンサーなど、入力として機能するセンサーがある。 これらのセンサーは、ユーザが単に向きを変えるかまたは装置を特定の方法で動かすことで何かを制御することを可能にできる。 他の状況では、ウェブコンテンツはカメラまたは他のセンサを介してユーザのジェスチャを解釈して機能を作動させることができる。 たとえば、デバイスを振ると”Undo”コマンドが発行されうる。また、穏やかな手の波（訳注：スワイプ動作に相当する）を使い一連のページを前後に移動しうる。 デバイスが（おそらく車いすの）固定マウントにあるため、または運動障害のために、ある障害者ユーザはこれらのデバイスセンサーを（まったく、または正確には不十分にしか）操作できない。機能は、他のまたは追加の活性化する手段が利用できるよう実装されなければならない。
-</p>
-            <p>さらに、あるユーザは、振戦（訳注：医学用語。手、頭、声帯、体幹、脚などの体の一部に起こる、不随意でリズミカルなふるえ。）または他の運動障害のために誤ってセンサを作動させる可能性がある。 ユーザは、このような偶然の機能の起動を防ぐよう動きによ起動をオフにする能力を持たなければならない。 アプリケーションは、ユーザがシステムレベルで動きの検出を無効に設定できるオペレーティングシステムをサポートすることでこの要求を満たすことができるであろう。</p>
+            <p>デバイスには、電話やタブレットデバイスの加速度センサーやジャイロセンサーなど、入力として機能するセンサーがある。 これらのセンサーは、利用者が単に向きを変えるか、又は装置を特定の方法で動かすことで何かを制御することを可能にできる。 他の状況では、ウェブコンテンツはカメラ、又は他のセンサーを介して利用者のジェスチャーを解釈して機能を作動させることができる。 たとえば、デバイスを振ると”Undo”コマンドが発行されうる。また、穏やかな手の波 (gentle hand wave) を使い一連のページを前後に移動しうる。【訳注：日本語訳の穏やかな手の波 (gentle hand wave) はスワイプ動作に相当する。】デバイスが (おそらく車いすの) 固定マウントにあるため、又は運動障害のために、ある利用者はこれらのデバイスセンサーを (まったく、又は正確には不十分にしか) 操作できない。機能は、他の又は追加の活性化する手段が利用できるよう実装されなければならない。</p>
```

Comment:
訳注スタイルの当て直し。

>Some users with disabilities are not able to operate these device sensors (either not at all, or not precisely enough) because the device is on a fixed mount (perhaps a wheelchair) or due to motor impairments. 

ある利用者はこれらのデバイスセンサーを (まったくないか、正確には十分ではないかのいずれかしか) 操作できない。